samples/analyze.js

```diff
@@ -301,6 +301,299 @@ async function analyzeVideoTranscription(gcsUri) {
   // [END video_speech_transcription_gcs]
 }
 
+async function analyzeTextGCS(gcsUri) {
+  //[START video_detect_text_gcs]
+  // Imports the Google Cloud Video Intelligence library
+  const Video = require('@google-cloud/video-intelligence').v1;
+  // Creates a client
+  const video = new Video.VideoIntelligenceServiceClient();
+
+  /**
+   * TODO(developer): Uncomment the following line before running the sample.
+   */
+  // const gcsUri = 'GCS URI of the video to analyze, e.g. gs://my-bucket/my-video.mp4';
+
+  const request = {
+    inputUri: gcsUri,
+    features: ['TEXT_DETECTION'],
+  };
+  // Detects text in a video
+  const [operation] = await video.annotateVideo(request);
+  const results = await operation.promise();
+  console.log('Waiting for operation to complete...');
+  // Gets annotations for video
+  const textAnnotations = results[0].annotationResults[0].textAnnotations;
+  textAnnotations.forEach(textAnnotation => {
+    console.log(`Text ${textAnnotation.text} occurs at:`);
+    textAnnotation.segments.forEach(segment => {
+      const time = segment.segment;
+      if (time.startTimeOffset.seconds === undefined) {
+        time.startTimeOffset.seconds = 0;
+      }
+      if (time.startTimeOffset.nanos === undefined) {
+        time.startTimeOffset.nanos = 0;
+      }
+      if (time.endTimeOffset.seconds === undefined) {
+        time.endTimeOffset.seconds = 0;
+      }
+      if (time.endTimeOffset.nanos === undefined) {
+        time.endTimeOffset.nanos = 0;
+      }
+      console.log(
+        `\tStart: ${time.startTimeOffset.seconds}` +
```

Comment:
Using `\t` is really hard to read.  Can we just use spaces instead?

Reply:
Fixed in latest commit.